app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Post;
+
+class PostController extends Controller
+{
+    public function store(Request $request)
```

Comment:
Doc block each method, here's an example - http://docs.phpdoc.org/getting-started/your-first-set-of-documentation.html#example